### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]
we may want to give ownership to the corresponding areas (in addition to test) but this works for me too

File: CODEOWNERS [hunk 1]
```diff
@@ -1,16 +1,6 @@
...
-/prow/cluster/jobs/                 @howardjohn @sdake @geeknoid @duderino @linsun
-/prow/cluster/jobs/istio/bots/      @howardjohn @sdake @geeknoid @duderino @linsun @ozevren
-/prow/cluster/jobs/istio/community/ @howardjohn @sdake @geeknoid @duderino @linsun
-/prow/cluster/jobs/istio/installer/ @howardjohn @sdake @geeknoid @duderino @linsun @costinm
-/prow/cluster/jobs/istio/istio.io/  @howardjohn @sdake @geeknoid @duderino @linsun @frankbu
-/prow/cluster/jobs/istio/istio/     @howardjohn @sdake @geeknoid @duderino @linsun @costinm @rshriram
-/prow/cluster/jobs/istio/operator/  @howardjohn @sdake @geeknoid @duderino @linsun @ayj @costinm @Laci21 @martonsereg @matyix @ostromart @rcernich @waynz0r
-/prow/cluster/jobs/istio/pkg/       @howardjohn @sdake @geeknoid @duderino @linsun @ozevren
-/prow/cluster/jobs/istio/proxy/     @howardjohn @sdake @geeknoid @duderino @linsun @mandarjog @PiotrSikora
-/prow/cluster/jobs/istio/test-infra @clarketm @cjwagner @fejta @Katharine @krzyzacy @michelle192837
```
Comment (fejta) [inline CODEOWNERS:15]: Can we leave test-infra as it was?

**Review by fejta — APPROVED** [review]
/hold

**Comment by geeknoid** [comment]
Sorry, due to some technical snafu, I've had to close this PR and reopen a fresh one.

**Pull request closed without merging**